### PR TITLE
WQP-1604  Updated wqp-etl-stewards to version 3.2.0

### PR DESCRIPTION
**Pull request by orosz-usgs**
Updated wqp-etl-core version used to 0.2.1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,13 +2,13 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>gov.acwi</groupId>
     <artifactId>wqp-etl-stewards</artifactId>
-    <version>3.2.0-SNAPSHOT</version>
+    <version>3.2.0</version>
```
Comment (mbucknell) [inline pom.xml:5]: Doesn't the build job for these update the version when the BUILD_TYPE is releases?

Comment (orosz-usgs) [reply pom.xml:5]: yes, I missed the bumpversion configuration.

Comment (orosz-usgs) [reply pom.xml]: @ayan-usgs  can you confirm that we do not need to update the version here?

Comment (ayan-usgs) [reply pom.xml]: https://github.com/NWQMC/etl-stewards/blob/master/.bumpversion.cfg#L17 and `dockerPipeline()` method should take of version incrementing.

**Review by ayan-usgs — APPROVED** [review]

**Pull request merged**